packages/server/src/server.ts

```diff
@@ -793,6 +793,10 @@ app.useRpc('eth_coinbase', async () => {
   return logAndHandleResponse('eth_coinbase', [], (requestDetails) => relay.eth().coinbase(requestDetails));
 });
 
+app.useRpcRegex(/^engine_.*$/, async () => {
```

Comment:
If the method name starts with `engine_` but is not part of the Ethereum JSON-RPC spec https://ethereum.github.io/execution-apis/api-documentation/, for example, `engine_foo`, do we want to return method not found or unsupported method?

Reply:
This was discussed and the reason for this was to avoid having not found if a future engine_ method is added. But if you believe this is incorrect behavior I can change it.

Reply:
Sounds about right, thanks for the clarification.